alluxio-python-library/alluxio/alluxio_file_system.py

```diff
@@ -278,6 +306,40 @@ def _create_session(self, concurrency):
         session.mount("http://", adapter)
         return session
 
+    def _load_file(self, worker_host, path):
+        try:
+            response = self.session.get(
+                self.LOAD_SUBMIT_URL_FORMAT.format(
+                    worker_host=worker_host,
+                    http_port=self.http_port,
+                    path=path,
+                ),
+            )
+            response.raise_for_status()
+            while True:
+                response = self.session.get(
+                    self.LOAD_PROGRESS_URL_FORMAT.format(
+                        worker_host=worker_host,
+                        http_port=self.http_port,
+                        path=path,
+                    ),
+                )
+                response.raise_for_status()
+                content = response.content
+                if b"SUCCEEDED" in content:
+                    print(content.decode('utf-8'))
+                    return True
+                if b"FAILED" in content:
+                    print(content.decode('utf-8'))
+                    return False
+                time.sleep(10)
```

Comment:
Do you feel it will be helpful to have a timeout? @ChunxuTang @jja725 how do we expect the user behavior of loading data? if load normally takes several hours, do we really expect users to call it from Alluxio python api? if it's small then it totally make sense

Reply:
agree we should have a timeout here to prevent user hang for long time.

Reply:
@jja725 @ChunxuTang  How about having different load operations?
- Submit_load
- Check_load_progress
- Sync_load with optional timeout parameter defined by user
Otherwise, if timeout, user won't know the exact load behavior, they don't have a way to check load status again

Reply:
agree with optional timeout